src/lib/djerba/render/database.py

```diff
@@ -0,0 +1,75 @@
+"""Interface with a CouchDB instance for JSON report documents"""
+
+import logging 
+import configparser 
+import requests
+import json
+import os
+
+import djerba.util.constants as constants
+import djerba.util.ini_fields as ini
+from datetime import datetime
+from djerba.util.logger import logger
+
+class Database(logger):
```

Comment:
For consistency with other djerba code, please use `lower_case_with_underscores` for class and method names: `Database` -> `database`, `UploadFile` -> `upload_file`